core/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -268,10 +268,6 @@ public Environment getEnvironment() {
      * Start the node. If the node is already started, this method is no-op.
      */
     public Node start() {
-        if (!lifecycle.moveToStarted()) {
```

Comment:
That was a mistake when I collapsed the two `start` methods back down to one. I will fix, thank you for noticing!